aula-02/exercicio-livro/styles.css

```
@@ -0,0 +1,271 @@
+
+  .open-book {
+    margin: 16px;
+    position: relative;
+  }
+
+  .open-book:before {
+    background-color: #8B4513;
+    border-radius: 4px;
+    bottom: -16px;
+    content: '';
+    left: -16px;
+    position: absolute;
+    right: -16px;
+    top: -16px;
+    z-index: -1;
+  }
+
+  .open-book:after {
+    background: linear-gradient(to right, transparent 0%,rgba(0,0,0,0.2) 46%,rgba(0,0,0,0.5) 49%,rgba(0,0,0,0.6) 50%,rgba(0,0,0,0.5) 51%,rgba(0,0,0,0.2) 52%,transparent 100%);
+    bottom: -16px;
+    content: '';
+    left: 50%;
+    position: absolute;
+    top: -16px;
+    transform: translate(-50%,0);
+    width: 64px;
+    z-index: 1;
+  }
+
+  .open-book > * {
+    column-count: 2;
+    column-gap: 96px;
+    position: relative;
+    z-index: 1;
+  }
+
+  .open-book header:before,
+  .open-book header:after,
+  .open-book footer:before,
+  .open-book footer:after {
+    background: #fff;
+    border-radius: 25%;
+    content: '';
+    height: 32px;
+    position: absolute;
+    z-index: -1;
+    width: calc(50% + 32px);
+  }
+
+  .open-book header:before,
+  .open-book footer:before,
+  .open-book footer:after {
+    border-top-left-radius: 0;
+  }
+
+  .open-book header:after,
+  .open-book footer:before,
+  .open-book footer:after {
+    border-top-right-radius: 0;
+  }
+
+  .open-book header:before,
+  .open-book header:after,
+  .open-book footer:after {
+    border-bottom-right-radius: 0;
+  }
+
+  .open-book header:before,
+  .open-book header:after,
+  .open-book footer:before {
+    border-bottom-left-radius: 0;
+  }
+
+  .open-book header:before,
+  .open-book header:after {
+    top: -42.4px;
+  }
+
+  .open-book header:before,
+  .open-book footer:before {
+    right: 50%;
+  }
+
+  .open-book header:before {
+    transform: rotate(-2deg);
+  }
+
+  .open-book header:after,
+  .open-book footer:after {
+    left: 50%;
+  }
+
+  .open-book header:after {
+    transform: rotate(2deg);
+  }
+
+  .open-book footer:before,
+  .open-book footer:after {
+    bottom: -42.4px;
+  }
+
+  .open-book footer:before {
+    transform: rotate(2deg);
+  }
+
+  .open-book footer:after {
+    transform: rotate(-2deg);
+  }
+
+  .open-book header > *:last-child,
+  .open-book footer > *:last-child {
+    text-align: right;
+  }
+
+  .open-book footer #page-numbers {
+    display: block;
+  }
+
+  .open-book .chapter-title {
+    font-size: 48px;
+  }
+
+  .open-book .chapter-title:before,
+  .open-book .chapter-title:after {
+    height: 0.125em;
+  }
+  .Text{
+    font: 3em serif;
+    position: relative;
+    font-weight: bold;
+  }
+  .img{
+    background: linear-gradient(#cedce7 0%,#596a72 50%);
+  }
+  .input{  
+    background: linear-gradient(#cedce7 0%,#596a72 50%);
+  }    
+  .p{
+    background: linear-gradient(#cedce7 0%,#596a72 50%);
+  }    
+  .yellow{    
+    background: linear-gradient(#e6f0a3 0%,#d2e638 50%,#c3d825 51%,#dbf043 100%);
+  } 
+  .box1{
+    background: linear-gradient(#fb83fa 0%,#e93cec 100%);
+  } 

```

Pra deixar mais organizado, sempre deixamos uma linha em branco em cada seletor, e vai um espaço entre a classe e a abertura das chaves {


Em vermelho como tu fez, em verde como eu to sugerindo.

```suggestion

  .yellow {    
    background: linear-gradient(#e6f0a3 0%,#d2e638 50%,#c3d825 51%,#dbf043 100%);
  } 

  .box1 {
    background: linear-gradient(#fb83fa 0%,#e93cec 100%);
  } 

```